massspecgym/runner.py

```diff
@@ -0,0 +1,244 @@
+import torch
```

Comment:
Adamo, could you move this file for example to `massspecgym/scripts/runnner_simulation.py`? We will try to merge it with `massspecgym/scripts/run.py` (equivalent file for the de novo and retrieval challenges) in the future (probably next month). If it is located in `massspecgym/runner.py` it may confuse people because it is designed for the simulation challenge at the moment.